src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -56,7 +61,8 @@ private void initializeDrivingVariants() {
         // treating it specially (separate from the other sources of Features in our FeatureManager).
         final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingVariantFile);
         if (VariantContext.class.equals(codec.getFeatureType())) {
-            drivingVariants = new FeatureDataSource<>(drivingVariantFile, (FeatureCodec<VariantContext, ?>)codec);
+            //This is the lookahead for the iteration of the driving data source
```

Comment:
Confusing comment -- reword as "This is the data source for the driving source of variants, which uses a cache lookahead of FEATURE_CACHE_LOOKAHEAD"